files/generate_vars.py

```diff
@@ -181,10 +180,8 @@ def _validate_output_file_exists(self, fname, log):
                         sys.exit(0)
```

Comment:
Is it me or the 2 lines are not properly indented?
sys.exit(0)
break

Reply:
seems okay. break belongs to outer if. if you exit there is no need for break after it.